### PR TITLE
Improves training instructions

### PR DESCRIPTION
**Pull request by emileighoutlaw**

### CODE REVIEW

File: _pages/qa/in-person-training.md [hunk 1]
```diff
@@ -1,52 +1,35 @@
 ---
 layout: "q-and-a"
 permalink: "/inperson-training/"
-title: "In-person Training"
+title: "Training and classes"
```
Comment (mbland) [inline _pages/qa/in-person-training.md]: Want to change the `permalink:` as well?

Comment (emileighoutlaw) [inline _pages/qa/in-person-training.md]: Yeah! is it just as easy as changing the permalink to training-and-classes?

Comment (mbland) [inline _pages/qa/in-person-training.md]: Yep, that's it!

**Comment by emileighoutlaw** [comment]
Done @mbland!

**Comment by emileighoutlaw** [comment]
@mbland any idea why it's now giving me a merge conflict?

**Comment by mbland** [comment]
Hey @emileighoutlaw, there was definitely a conflict on that line with what you'd written before in commit 534d5cd95ec0be40a906a085bae6057912bc1a0c. Take a look and make sure this is the version of the text that you want?

**Pull request merged**